types/ember/index.d.ts

```diff
@@ -2113,7 +2113,10 @@ declare module 'ember' {
                 options?: { path?: string; resetNamespace?: boolean },
                 callback?: (this: RouterDSL) => void
             ): void;
-            mount(name: string): void;
+            mount(
+                name: string,
+                options?: { as?: string, path?: string; resetNamespace?: boolean, engineInfo?: any }
+            ): void
```

Comment:
tslint is failing because of a missing semicolon here

Reply:
ops! Good eyes!